map-downloader-android/app/src/main/AndroidManifest.xml

```diff
@@ -38,12 +38,12 @@
         <service
             android:name="com.here.android.mpa.service.MapService"
             android:label="HereMapService"
```

Comment:
I would have to check mobile SDK sources but better be safe than sorry. I'll add it.